lib/api_test.go

```diff
@@ -439,5 +441,85 @@ func TestGetTransactionsHandler(t *testing.T) {
 		assert.Equal(t, bt.Hash, receivedBts[i].Hash, "hash is not same")
 		i++
 	}
+}
+
+func TestNodeHandler(t *testing.T) {
+
+	var wg sync.WaitGroup
+	wg.Add(1)
+
+	var validator *sebakcommon.Validator
+	kp, _ := keypair.Random()
+	endpoint, err := sebakcommon.NewEndpointFromString("dummy")
+	checkError(t, err)
+	validator, err = sebakcommon.NewValidator(kp.Address(), endpoint, "")
+	checkError(t, err)
+	ctx := context.WithValue(context.Background(), "currentNode", validator)
+
+	router := mux.NewRouter()
+	router.HandleFunc(GetNodePattern, GetNodeHandler(ctx)).Methods("GET")
+
+	ts := httptest.NewServer(router)
+	defer ts.Close()
+
+	// Do a Request
+	url := ts.URL + "/node"
+
+	req, _ := http.NewRequest("GET", url, nil)
+	req.Header.Set("Accept", "text/event-stream")
+
+	resp, _ := http.DefaultClient.Do(req)
+	reader := bufio.NewReader(resp.Body)
+
+	var dummyValidatorIDs [10]string
+	for i := 0; i < 10; i++ {
+		key, _ := keypair.Random()
+		dummyValidatorIDs[i] = key.Address()
+	}
 
+	// Do stream Request to the Server
+	go func() {
+		for i := 0; i < 5; i++ {
+			line, _ := reader.ReadBytes('\n')
+			var v sebakcommon.Validator
+			json.Unmarshal(line, &v)
+
+			assert.Equal(t, validator.Alias(), v.Alias(), "alias is not same")
+			assert.Equal(t, validator.Address(), v.Address(), "address is not same")
+			assert.Equal(t, validator.Endpoint().String(), v.Endpoint().String(), "endpoint is not same")
+			if i != 0 && !v.HasValidators(dummyValidatorIDs[i-1]) {
+				t.Errorf("%s validator is not exist", dummyValidatorIDs[i])
+			}
+		}
+		resp.Body.Close()
+		wg.Done()
+	}()
+	for i := 0; i < 10; i++ {
+		var newValidator *sebakcommon.Validator
+		endpoint, _ := sebakcommon.NewEndpointFromString(sebakcommon.GetUniqueIDFromUUID())
+		newValidator, _ = sebakcommon.NewValidator(dummyValidatorIDs[i], endpoint, "")
+		validator.AddValidators(newValidator)
+	}
+
+	wg.Wait()
+	// No streaming
```

Comment:
I think, it would be better to split `No streaming` to the other test case for readability.
Like below :)
```
func TestNodeHandlerWithStreamRequest(t *testing.T) {
and
func TestNodeHandlerWithNoStreaming(t *testing.T) {
```